Contributing.md

```diff
@@ -46,10 +40,25 @@ Before diving in, follow the [Manual Installation Instructions](docs/Manual%20In
 * Add new features:
   * If you would like to become involved in the development effort, please hop
    on our [OFTC channel at #activecm](https://webchat.oftc.net/?channels=activecm)
-   and chat about what is currently being worked on.
+   and chat about what is currently being worked on
 
-All of these tasks ultimately culminate in a pull request being issued,
-reviewed, and merged. 
+### Running Static Tests
+* Golint
+  * Install [golint](https://github.com/golang/lint)
+  * Run `golint ./...` from the root RITA directory
```

Comment:
I'd recommend changing this to `golint ./... | grep -v '^vendor/'` as the command as written on my system will also look at the `vendor` directory.

Reply:
Added in the grep -v command as suggested.